components/validator/cmd/main.go

```diff
@@ -28,6 +29,13 @@ func main() {
 	app.Description = "Service for generating and submitting L2 output checkpoints to the L2OutputOracle contract as an L2 Output Submitter, " + "detecting and correcting invalid L2 outputs as a Challenger to ensure the integrity of the L2 state."
 
 	app.Action = curryMain(Version)
+	app.Commands = []cli.Command{
+		{
+			Name:        "balance",
+			Subcommands: balance.Subcommands,
+		},
+	}
```

Comment:
I think it looks better that `deposit`, `withdraw` and `unbond` subcommands are declared here.